docs/Readme.md

```diff
@@ -25,6 +25,21 @@ Here is a few short video clips that can quickly explain DLWorkspace. Note the P
 * [Single Ubuntu Computer](deployment/On-Prem/SingleUbuntu.md)
 
 ## [Frequently Asked Questions](KnownIssues/Readme.md)
+* How to import the credential so that each node is accessible? <br/>

```

Comment:
Thanks for the contribution. However, 

1. All the features discussed has already been covered in existing documentation:
https://github.com/Microsoft/DLWorkspace/tree/alpha.v1.4/docs/deployment/Scripts
https://github.com/Microsoft/DLWorkspace/blob/alpha.v1.4/docs/deployment/Backup.md

2. It is generally not a good idea to discuss details in the high level document src/Readme.md.

I will close this PR.